### PR TITLE
Use standard dropdown control for QR codes in results list.

### PR DESCRIPTION
**Pull request by EreMaijala**
Adds a template for the QR code to avoid duplication of code.

### CODE REVIEW

File: themes/bootstrap3/templates/RecordDriver/DefaultRecord/controls/qrcode.phtml
Comment (demiankatz) [inline themes/bootstrap3/templates/RecordDriver/DefaultRecord/controls/qrcode.phtml]: Are you intentionally omitting the alt here? (I realize that an alt of "QR Code" isn't especially useful... but I wonder if it's better than nothing... also, we should be consistent across different QR Code displays, and the alt tag is still present in core.phtml and collection-info.phtml).

File: themes/bootstrap3/js/common.js
Comment (demiankatz) [inline themes/bootstrap3/js/common.js:626]: The reason this wasn't working is that the qrcode translation strings were not being passed through in layout/js-translations.phtml. Would it be better to fix that rather than to remove the hide translation? (I don't have strong feelings one way or another, but just raising the question).

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks! This looks pretty good, but I had a couple of questions before merging...

File: themes/bootstrap3/js/common.js
Comment (EreMaijala) [reply themes/bootstrap3/js/common.js:626]: I decided to remove it for consistency with all other dropdowns that don't change the text. But it could be added back if that's better.

File: themes/bootstrap3/templates/RecordDriver/DefaultRecord/controls/qrcode.phtml
Comment (EreMaijala) [reply themes/bootstrap3/templates/RecordDriver/DefaultRecord/controls/qrcode.phtml]: It was intentional, but you're right in that it was also inconsistent. I restored the alt text here.

File: themes/bootstrap3/js/common.js
Comment (demiankatz) [reply themes/bootstrap3/js/common.js:626]: Makes sense to me, but I'm going to ask @crhallberg for an additional review in case he has any further thoughts or opinions. :-)

**Review by demiankatz — APPROVED** [review]

File: themes/bootstrap3/js/common.js
Comment (crhallberg) [reply themes/bootstrap3/js/common.js:626]: I'm of the opinion that the text should change, especially since we already have the translation strings. In the future, we can make the other toggles consistent.

**Review by crhallberg — APPROVED** [review]
I'll leave it up to you and DK about the translation strings, but this is definitely a huge improvement. Thank you!

File: themes/bootstrap3/js/common.js
Comment (demiankatz) [reply themes/bootstrap3/js/common.js:626]: Thanks, @crhallberg. If @EreMaijala doesn't mind putting back the text switching, it would greatly reduce the number of files touched by this PR and reduce the behavior changes as well. But if he feels strongly that it's better this way, he already has approvals from both of us and is free to hit the merge button. I'll leave the final decision up to him! :-)

Comment (EreMaijala) [reply themes/bootstrap3/js/common.js:626]: The problem with text switching, as I see it, is that it makes the aria attributes a bit weird. If you have `aria-expanded="true"` on a toggle that says "Hide QR Code", it just doesn't make a whole lot of sense. Maybe it would be best to remove the verb and just display e.g. "QR Code ▼"? We already have a translation for "QR Code", so no new language strings would be needed.

Comment (demiankatz) [reply themes/bootstrap3/js/common.js:626]: @EreMaijala, that sounds like a great compromise to me, and I think it probably lets us get rid of a second translation string. :-)

Comment (EreMaijala) [reply themes/bootstrap3/js/common.js:626]: Done. :)

**Comment by demiankatz** [comment]
Looks good to me! I'm running the test suite to be safe, and I'll merge if it passes.

**Pull request merged**
